templates/components/scenarios.tmpl

```diff
@@ -183,7 +183,7 @@
                       <% if (step.image) { %>
                         <div id="info<%= scenarioIndex %><%= stepIndex %>-image" class="scenario-step-collapse collapse">
                           <a class="screenshot" href="<%= step.image %>" target="_blank">
-                    <img class="screenshot" style="height:100%;width:100%;" id="my_images" src="<%= step.image %>"/>
```

Comment:
Removal was not needed because now big images exceed the width of the window. But this triggers me that `id="my_images"` needs to be replaced with a `class` because we can have multiple screenshots in a feature

Reply:
we can add on style.css

``
.screenshot {
  width: 100%;
  height: 100%;
}
``

for multiple screenshot must be change step.image into an array.